### PR TITLE
Add Custard Insurance Adjusters

### PR DESCRIPTION
**Pull request by DivyaB1**
Hi,

I'm adding Custard Insurance Adjusters - https://custard.com/

Thanks,

### CODE REVIEW

**Comment by AntiCompositeNumber** [comment]
Hrm. The definition of [`office=insurance`](https://wiki.openstreetmap.org/wiki/Tag:office%3Dinsurance) is a bit of a mess, but I'm not sure an insurance adjuster fits it. However, every insurance adjuster I can find mapped (not many) is tagged `office=insurance`.

**Comment by DivyaB1** [comment]
> Hrm. The definition of [`office=insurance`](https://wiki.openstreetmap.org/wiki/Tag:office%3Dinsurance) is a bit of a mess, but I'm not sure an insurance adjuster fits it. However, every insurance adjuster I can find mapped (not many) is tagged `office=insurance`.

Hi,
`office=insurance is the most relevant tag I could find. Do you have other suggestions for me to update it?

Thanks,

**Comment by AntiCompositeNumber** [comment]
[A discussion](https://osmus.slack.com/archives/C2VJAJCS0/p1699388237539229) in the [OSM US Slack](https://openstreetmap.us/get-involved/slack/) suggests using `office=insurance_adjuster`.

File: data/brands/office/insurance_adjuster.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "properties": {
+    "path": "brands/office/insurance_adjuster",
+    "preserveTags": ["^name"],
```
Comment (Cj-Malone) [inline data/brands/office/insurance_adjuster.json]: I don't think we need this at this stage.

```suggestion
```

**Review by Cj-Malone — APPROVED** [review]

**Pull request merged**